spec-main/api-app-spec.ts

```diff
@@ -973,6 +973,31 @@ describe('app module', () => {
     });
   });
 
+  describe('getApplicationInfoForProtocol()', () => {
```

Comment:
We can use `ifdescribe(process.platform !== 'linux')` here and remove the skip conditionals from individual tests

Reply:
thank you